### ISSUE TITLE
Chai 3.4.0

### ISSUE THREAD

**Issue by SteveEdson** [chaijs/chai-fs#10]
Any chance this could be updated to work with Chai 3.4.0 please?

Thanks

**Comment by scheibinger** [comment]
:+1:

**Comment by yoannmoinet** [comment]
:+1:

**Comment by mmun** [comment]
What's wrong? It seems to be working for me with chai@3.4.1, other than this warning:

```
npm WARN chai-fs@0.1.0 requires a peer of chai@>= 1.6.1 < 2 but none was installed.
```

**Comment by dmartinzar** [comment]
It may technically work with Chai 3, but unless the peer dependency is fixed that warning becomes an error for `npm shrinkwrap`:

```
npm ERR! Linux 3.10.0-327.3.1.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "shrinkwrap" "--dev"
npm ERR! node v5.4.0
npm ERR! npm  v3.5.3

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! peer invalid: chai@>= 1.6.1 < 2, required by chai-fs@0.1.0
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

**Comment by jhnns** [comment]
I don't know if all assertions are working, but in my setup `chai-fs` is working just fine. Would be good to get rid of the peer dependency warning. :+1:

**Comment by jhnns** [comment]
Btw: It's also a problem because [CI builds are failing](https://travis-ci.org/peerigon/extract-loader) just because `npm install` exits with an error code. So it's not just a cosmetic problem.

**Comment by Bartvds** [comment]
Sorry guys, I've crashed out of the loop on node and have no idea what is current on grunt, npm and node. At this point I cannot push to npm at the risk of nuking whoever still has this working.

I think someone who knows what they are doing should take ownership, sort this out and be a better steward.

If one (or more) of you are up for it please send me your npm and github handles and I will transfer repos and npm ownership soonest.

**Comment by Bartvds** [comment]
Quick update: @keithamus offered to adopt this module in the Chaijs organisation. Probably the ideal solution so we're proceeding on that right now.

**Comment by SteveEdson** [comment]
Excellent news for everyone, thanks. Thanks for all the work @Bartvds :+1:

**Comment by jhnns** [comment]
@Bartvds @keithamus thank you!

**Comment by meeber** [comment]
I think this was resolved via #8. Can we close this? @keithamus

**Comment by keithamus** [comment]
We can. There is some work to do to release this though. It's on my to-do list 😉
